run.sh

```diff
@@ -1,67 +0,0 @@
-#!/bin/bash
-#
-# Miscellaneous scripts.
```

Comment:
Ok, great, then let's check it into that branch.

On Thu, Feb 4, 2016 at 5:47 PM, andychu <notifications@github.com> wrote:

> In run.sh <https://github.com/google/rappor/pull/69#discussion_r51967296>:
>
> > @@ -1,67 +0,0 @@
> > -#!/bin/bash
> > -#
> > -# Miscellaneous scripts.
>
> It's useful for publishing to github pages, which we are using. Please
> restore the file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/rappor/pull/69/files#r51967296>.
>